src/components/StaffComments/StaffComments.vue

```diff
@@ -99,19 +100,13 @@ export default class StaffComments extends Mixins(DateMixin, FilingMixin) {
     textarea: FormIF
   }
 
-  /** Axios instance prop. */
-  @Prop({ required: true })
-  readonly axios: any
-
-  /** Business ID prop. */
-  @Prop({ required: true })
-  readonly businessId: string
+  @Prop({ required: true }) readonly axios: any
+  @Prop({ required: true }) readonly businessId: string
+  @Prop({ default: 33 }) readonly nudgeTop: number
+  @Prop({ default: 20 }) readonly nudgeLeft: number
```

Comment:
Since these nudges are specific to how the main button is styled (ie, in whichever app imports this), I made them configurable. These defaults work nicely in our app :)